### PR TITLE
New template: cisco_ios_show_power_inline.textfsm

### PR DESCRIPTION
**Pull request by JKLs12**
Added new cisco template cisco_ios_show_power_inline.textfsm for PoE status

### CODE REVIEW

File: ntc_templates/templates/cisco_ios_show_power_inline.textfsm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  ^\s*$$
+  ^. -> Error
+
+Interfaces
+  ^\s*${Interface}\s+${Admin}\s+${Oper}\s+${Power}\s+${Device}\s+${Class}\s+${Max}\s*$$ -> Record
+  ^Totals.*
+  ^\s+\(Watts\)
+  ^-+
+  ^\s*$$
+  ^. -> Error
```
Comment (jmcgill298) [inline ntc_templates/templates/cisco_ios_show_power_inline.textfsm:24]:
```suggestion
  ^. -> Error

```

**Review by jmcgill298 — APPROVED** [review]

**Comment by jmcgill298** [comment]
@JKLs12 there are helper files to clean up yaml files: https://ntc-templates.readthedocs.io/en/latest/dev/dev_environment/#cli-helper-commands

File: ntc_templates/templates/cisco_ios_show_power_inline.textfsm [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Value Interface (\S+)
+Value Admin (\S+)
+Value Oper (on|off)
+Value Power (\d+\.\d+)
+Value Device (\S+(\s\S+)*)
+Value Class (\d|n/a)
+Value Max (\d+\.\d+)
```
Comment (jmcgill298) [inline ntc_templates/templates/cisco_ios_show_power_inline.textfsm:7]:
```suggestion
Value INTERFACE (\S+)
Value ADMIN_STATUS (\S+)
Value OPERATIONAL_STATUS (on|off)
Value POWER (\d+\.\d+)
Value DEVICE (\S+(\s\S+)*)
Value CLASS (\d|n/a)
Value MAX (\d+\.\d+)

```
We standardize on using all caps for capture group names for ease of identifying, and started standardizing some of the names across all templates

File: ntc_templates/templates/cisco_ios_show_power_inline.textfsm [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ^Interface\s+Admin\s+Oper\s+Power\s+Device\s+Class\s+Max -> Interfaces
+  ^Module
+  ^\d+
+  ^-+
+  ^\s+\(Watts\)
+  ^\s*$$
+  ^. -> Error
+
+Interfaces
+  ^\s*${Interface}\s+${Admin}\s+${Oper}\s+${Power}\s+${Device}\s+${Class}\s+${Max}\s*$$ -> Record
```
Comment (jmcgill298) [inline ntc_templates/templates/cisco_ios_show_power_inline.textfsm:19]:
```suggestion
  ^\s*${INTERFACE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${POWER}\s+${DEVICE}\s+${CLASS}\s+${MAX}\s*$$ -> Record

```

File: tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.raw [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Gi1/0/15  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/16  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/17  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/18  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/19  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/20  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/21  off    off        0.0     n/a                 n/a   30.0
+Gi1/0/22  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/23  auto   off        0.0     n/a                 n/a   30.0
+Gi1/0/24  auto   off        0.0     n/a                 n/a   30.0
```
Comment (jmcgill298) [inline tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.raw:32]:
```suggestion
```

File: tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.raw [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+--------- ------ ---------- ------- ------------------- ----- ----
+Gi1/0/1   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/2   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/3   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/4   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/5   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/6   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/7   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/8   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
+Gi1/0/9   auto   on         30.0    AIR-AP1562I-E-K9    4     30.0
```
Comment (jmcgill298) [inline tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.raw:17]:
```suggestion
```
There is a lot of duplicate type data, just trim it down for ease of updating

File: tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.yml [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
+- Admin: "auto"
+  Class: "n/a"
+  Device: "n/a"
+  Interface: "Gi1/0/24"
+  Max: "30.0"
+  Oper: "off"
+  Power: "0.0"
```
Comment (jmcgill298) [inline tests/cisco_ios/show_power_inline/cisco_ios_show_power_inline.yml:170]:
```suggestion
parsed_sample:
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/1"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/2"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/3"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "4"
  device: "C9120AXE-E"
  interface: "Gi1/0/10"
  max: "30.0"
  operational_status: "on"
  power: "23.2"
- admin_status: "auto"
  class: "4"
  device: "AIR-AP1562I-E-K9"
  interface: "Gi1/0/11"
  max: "30.0"
  operational_status: "on"
  power: "30.0"
- admin_status: "auto"
  class: "n/a"
  device: "n/a"
  interface: "Gi1/0/12"
  max: "30.0"
  operational_status: "off"
  power: "0.0"
- admin_status: "auto"
  class: "n/a"
  device: "n/a"
  interface: "Gi1/0/13"
  max: "30.0"
  operational_status: "off"
  power: "0.0"
```

ntc-templates will convert all capture groups to lowercase

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Comment by JKLs12** [comment]
@jmcgill298 Thanks for your help and patience :)

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**